app/controllers/clients_controllers/post_login.py

```diff
@@ -15,16 +18,18 @@ def post_login():
             password=str,
             optional=[]
         )
-        
+
         user = Client.query.filter_by(email=clean_data["email"]).first_or_404(
             description={"msg": "user not found"})
 
-        if not user.check_password(clean_data["password"]):
+        if not user.verify_password(clean_data["password"]):
             return {"msg": "wrong password"}, HTTPStatus.FORBIDDEN
 
         token = create_access_token(
             identity=user, expires_delta=Config.JWT_ACCESS_TOKEN_EXPIRES)
 
         return {"access token": token}
     except werkzeug.exceptions.NotFound as e:
-        return e.description, HTTPStatus.NOT_FOUND
\ No newline at end of file
+        return e.description, HTTPStatus.NOT_FOUND
```

Comment:
Só tem que verificar se esse `e.description` que vem do `werkzeug` está vindo de acordo com a mensagem que está no Product Back log.

Reply:
O erro foi especificado na linha 27, identico ao do backlog, quanto ao erro do no body usei um padrão, se quando for especificado estiver diferente eu mudo.